.changes/next-release/bugfix-32139a16-d00e-43e2-aab8-a356008abfd1.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Fix telemetry logging for new Amazon Q Code Transform telemetry updates"
```

Comment:
non-userfacing so does not need a changelog

Reply:
Thanks for letting me know. Should I delete?

Reply:
happy to keep, doesn't hurt